### DISCUSSION TITLE
General Donation - Error should dispay when Entering zero

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#110] [Feedback]
![image](https://user-images.githubusercontent.com/101787508/159403512-ef2455c2-c4b3-4263-8a34-540c88d26b4a.png)

When zero is entered and we click on donate then it should show an error to indicate the problem to user.
Currently there is no response which leads to think that the button is not working.

2 suggested actions

1-Add a small animation or change in state of button on click(PC) or press(mobile) to indicate that button has actually been pressed.
2 - Add an error message if invalid value is entered example zero or someone enters a text value.

**Comment by smsalmanshahAN** [comment]
The only text character it accepts is e for exponent and it is giving an error if wong format is used for exponents or when just e is entered.
So the text characters are covered. 
Zero value also needs to indicate an error

**Comment by an-nadaa** [comment]
Thank you for reporting this I will use a more thoughtful validation library to fix the aforementioned problems probably this one: https://github.com/vuelidate/vuelidate as it supports Vue 2 and 3 and doesn't force us to use custom components
